portal-ecs.yml

```diff
@@ -292,9 +315,11 @@ Resources:
   ElasticSearchStack:
     Type: AWS::CloudFormation::Stack
     Properties:
-      TemplateURL: https://s3.amazonaws.com/concord-devops/cloud-formation/elastic-search.yml
+      TemplateURL: !Sub "${NestedTemplatesFolder}/elastic-search.yml"
       Parameters:
         ClusterSecurityGroupId: { 'Fn::ImportValue': !Sub '${ClusterStackName}-ClusterSecurityGroupId' }
+        VpcId: !Ref 'VpcId'
```

Comment:
`{ 'Fn::ImportValue': !Sub '${ClusterStackName}-ClusterVpcId' }` should be used here.